### DISCUSSION TITLE
Solidus_auth_devise : "You must confirm your account before continuing." (Admin)

### DISCUSSION THREAD

**Discussion by joyboy67** [solidusio/solidus#4632] [Troubleshooting] [answered]
Hello, I added `Spree::Auth::Config[:confirmable] = true` to `initializers/spree.rb` for enable confirmable module, but now I can't access to the admin panel : "You must confirm your account before continuing.",

I tried to edit the admin user by the rails console
```
u = Spree::User.find(1)
u.confirmed_at
# nil
u.confirmed_at = 1
u.save
```
but it's not working, if anyone has an idea i'm a taker :)

**Comment by waiting-for-dev** [comment, marked as answer]
Hey @joyboy67, you can just run `u.confirm` from the console 🙂

**Reply by joyboy67** [reply]
Thank you @waiting-for-dev it works perfectly :)
